src/js/components/LoginPage/Login.scss

```diff
@@ -0,0 +1,5 @@
+.login form {
```

Comment:
empty line at the end of file

Reply:
fixed thanks. Can I ask why it is necessary